src/api/java/com/minecolonies/api/entity/CustomGoalSelector.java

```diff
@@ -89,7 +85,7 @@ public CustomGoalSelector(@NotNull final IProfiler profiler)
         super(profiler);
         this.profiler = profiler;
         super.goals = this.goals;
-
+        super.profiler = this.profiler;
```

Comment:
twice?

Reply:
ye for each constructor

Reply:
Ahh okay, didn't see it was the next constructor in this small review window =D